front/src/modules/activities/timeline/components/TimelineActivity.tsx

```diff
@@ -166,9 +166,7 @@ export function TimelineActivity({ activity }: OwnProps) {
                 onCompletionChange={completeTask}
               />
               <StyledCardContent>
-                <OverflowingTextWithTooltip
-                  text={body ? body : '(No content)'}
-                />
+                <OverflowingTextWithTooltip text={body ? body : ''} />
```

Comment:
I would do {body && <OverflowingTextWithTooltip text={body ? body : ''} />}

Reply:
Oh yeah i just missed it.